tests/test_appsettings.py

```diff
@@ -25,15 +25,15 @@ def setUp(self):
         self.message_no_attr = "has no attribute '%s'"
 
     def test_setting(self):
-        setting = appsettings.Setting()
-        assert setting.name == ""
-        assert setting.full_name == ""
+        setting = appsettings.Setting(name='simple')
+        assert setting.name == 'simple'
+        assert setting.full_name == 'SIMPLE'
```

Comment:
Cannot settings be empty anymore?

Reply:
The `AttributeError` now contains only the name of the missing attribute. With empty name of the settings, the `AttributeError` contains only an empty string. That can't be properly checked in test below.